src/pages/Login/LoginForm.js

```diff
@@ -47,10 +47,10 @@ const LoginForm = () => {
                 })
             )
 
-            setEmail('')
-            setPassword('')
-            setEmailIsTouch(false)
-            setPasswordIsTouch(false)
+            // setEmail('')

```

Comment:
Not sure why these are commented out but maybe if we don't need them anymore we should delete them.